From b2eebdb7f943099bb8c68bedddee00b0781e3fa4 Mon Sep 17 00:00:00 2001
From: Adithya R <gh0strider.2k18.reborn@gmail.com>
Date: Sun, 6 Jun 2021 19:37:39 +0530
Subject: [PATCH 9/9] sweet: overlay: Define OEM fast charge sysfs node

 * this forces android to display "charging rapidly" whenever our
   proprietary 33w charger is connected, based on the value of
   this node (0/1)

Change-Id: I2a36ccb62621672e480e4b25a335410044fbb372
Signed-off-by: basamaryan <basam.aryan@gmail.com>
---
 .../frameworks/base/core/res/res/values/config.xml | 11 +++++++++++
 1 file changed, 11 insertions(+)
 create mode 100644 overlay-lineage/frameworks/base/core/res/res/values/config.xml

diff --git a/overlay-lineage/frameworks/base/core/res/res/values/config.xml b/overlay-lineage/frameworks/base/core/res/res/values/config.xml
new file mode 100644
--- /dev/null
+++ b/overlay-lineage/frameworks/base/core/res/res/values/config.xml
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     SPDX-FileCopyrightText: 2024 The LineageOS Project
+     SPDX-License-Identifier: Apache-2.0
+-->
+<resources>
+    <!-- Each entry is a sysfs path, delimiter, and active value. -->
+    <string-array name="config_oemFastChargerStatusPaths" translatable="false">
+        <item>/sys/class/power_supply/bms/fastcharge_mode;1</item>
+    </string-array>
+</resources>
-- 
2.53.0
